src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/ContentHelper.Common.cs

```diff
@@ -16,7 +19,7 @@ internal static class ContentHelper
         #region Internal Methods
 
         // ContentType may not exist in response header.  Return null if not.
-        internal static string GetContentType(HttpResponseMessage response) => response.Content.Headers.ContentType?.MediaType;
+        internal static string? GetContentType(HttpResponseMessage? response) => response?.Content.Headers.ContentType?.MediaType;
```

Comment:
Why is `response` checked for null? It seems it is never null.

Reply:
Probably leftover from a reverted change in a previous commit